src/js/letters/reducers/index.js

```diff
@@ -23,16 +27,34 @@ function letters(state = initialState, action) {
       // the various error scenarios
       return set('lettersAvailable', false, state);
     case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS':
+      // Create object for which options to include in post for summary letter
+      // Default all options to true so they appear checked in the UI
+      options = mapValues(action.data.data.attributes.benefitInformation, (value) => {
```

Comment:
What this section is doing is mapping through the list of options we get in the response from EVSS and putting all of those in a new object called `optionsToInclude`, which will store a boolean value for each option that will reflect whether the option is selected in the UI (`true`) or not (`false`).

Where it gets tricky is how the front-end handles concealing options that are sent to us by EVSS that we don't want to show in the UI, either because they're not eligible for those options (since the value is sent to us as `false`) or because the option is actually for the dependent summary letter (this shouldn't ever happen in real life, but you never know). What I do below is check if the value for each option is `!false`; if so I put it in `optionsToInclude`, defaulted to `true`. If the value is `false`, I put it in `optionsToInclude` as `false`. Hopefully that makes sense!